geowebcache/azureblob/src/main/java/org/geowebcache/azure/AzureBlobStoreData.java

```diff
@@ -0,0 +1,190 @@
+/**
+ * This program is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU Lesser General Public License as published by the Free Software Foundation, either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * <p>You should have received a copy of the GNU Lesser General Public License along with this
+ * program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author Fernando Mino, GeoSolutions, Copyright 2019
+ */
+package org.geowebcache.azure;
+
+import java.net.InetSocketAddress;
+import java.net.Proxy;
+import org.geowebcache.GeoWebCacheTypedEnvironment;
+
+/**
+ * Azure Blobstore type-resolved data from a {@link AzureBlobStoreInfo} using enviroment variables
+ * if enabled.
+ */
+public class AzureBlobStoreData {
```

Comment:
Type safe bean, yep, I would have likely done the same. Any actual need to make this class public, though?

Reply:
Right, not public makes sense.